opencti-platform/opencti-front/src/private/components/analyses/reports/ReportDetails.jsx

```diff
@@ -164,15 +199,17 @@ const ReportDetailsComponent = (props) => {
             xs={6}
             style={{ minHeight: 200, maxHeight: height }}
           >
-            <EntityStixCoreRelationshipsHorizontalBars
-              title={t('Entities distribution')}
-              variant="inEntity"
+            <StixRelationshipsHorizontalBars
+              isWidget={false}
               fromId={report.id}
-              toTypes={['Stix-Core-Object']}
+              startDate={null}
+              endDate={null}
               relationshipType="object"
-              field="entity_type"
-              seriesName={t('Number of entities')}
-              isTo={true}
+              dataSelection={entitiesDidstributionDataSelection}
+              parameters={'Entities distribution'}
```

Comment:
parameters is not an object?

Reply:
judging from the use of `parameters` [here](https://github.com/OpenCTI-Platform/opencti/blob/c394cc812d99db764aba933376cd0089990eba1a/opencti-platform/opencti-front/src/private/components/common/stix_relationships/StixRelationshipsHorizontalBars.jsx#L166) and [there](https://github.com/OpenCTI-Platform/opencti/blob/c394cc812d99db764aba933376cd0089990eba1a/opencti-platform/opencti-front/src/private/components/common/stix_relationships/StixRelationshipsHorizontalBars.jsx#L184) you probably
want to set `parameters={{ title: 'Entities distribution' }}`